pages/authCallbackOAuth2/authCallbackOAuth2.js

```diff
@@ -49,8 +49,8 @@ router.get('/', function (req, res, next) {
     }
     const params = [
       identity.email, // uid
-      identity.email, // name
-      null, // uin
+      identity.name || identity.email, // name (use email if name is not present)
+      identity.sub, // uin
```

Comment:
Do you have any idea what `sub` looks like? For example, is it a 256-character monstrosity that we shouldn't try and render on pages?

Reply:
The docs say at most 255 characters, but the example has ~30 digits. I imagine it's closer to the latter, but I don't have a simple way to test.

Reply:
May not be a bad idea to render the UIN with a max-width and ellipsis overlay anyway, though (in a different PR).

Reply:
Agreed.